frontend/src/pages/About.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+<div class="bg-white dark:bg-gray-900">
+    <!-- Intro section -->
+  <section class="text-gray-600 body-font dark:text-gray-400">
+  <div class="container flex flex-col items-center px-5 mx-auto py-14 md:flex-row">
+    
+    <div class="flex flex-col items-center text-center lg:flex-grow md:w-1/2 lg:pl-24 md:pl-16 md:items-start md:text-left">
+      <h1 class="mb-4 font-bold text-9xl sm:text-4xl text-rainbow">Rainbow</h1>
+      <p class="mb-8 leading-8 sm:w-full">Dashboard for DEX/CEX options trading highlighting market opportunities to ease strategies decision. To achieve this goal, Rainbow continuously scans CEX/DEX/DeFi market data.
+          This project has been initiated by <a href="https://teal.finance">Teal.Finance</a> and has already won two prices during the Ethereum's EthGlobal Hackathon.
+      </p>
+    </div>
+    <div class="w-1/2 mb-10 md:mb-0">
+      <img class="object-cover object-center w-3/4 rounded" alt="hero" src="@/assets/dashboard.svg">
+    </div>
+  </div>
+</section>
```

Comment:
Could you please format the file to indent the closing tags on opening tags: actually it's hard to read

Reply:
I format it by Beautify.